experimental/GroebnerWalk/src/GroebnerWalk.jl

```diff
@@ -0,0 +1,22 @@
+module GroebnerWalk
+using Oscar
+
+include("markedGB.jl")
+include("common.jl")
+
+include("standard_walk.jl")
+include("generic_walk.jl")
+include("perturbed_walk.jl")
+
+import Oscar: weight_ordering, ZZRingElem, MonomialOrdering, ZZMatrix, IdealGens
+import Oscar.Orderings: MatrixOrdering, _support_indices
+
+export groebner_walk
+
+function __init__()
+    add_verbosity_scope(:groebner_walk)
+
+    set_verbosity_level(:groebner_walk, 0)
```

Comment:
```suggestion
```
this is not needed, see e.g. https://github.com/oscar-system/Oscar.jl/blob/344710c8292b58fd42fca71c67278e1a6d20fdbf/src/Oscar.jl#L143-L179 for the place where the other scopes are introduced